index.html

```diff
@@ -278,7 +287,7 @@ <h4 class="contact-title">Contact me</h4>
         ></textarea>
         <button class="fbutt">Get It Touch</button>
       </form>
+    </div>
```

Comment:
- [ ] You should stick to the design as much as possible. Please fix the spacing and also the elements of the form are centered on the page including the Get in touch button.
- [ ] The background is missing please fix this issue also.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213681399-50338be9-e5b8-46d1-bc9c-1d640f958c44.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213681435-bccbebcc-aa85-4107-887f-f246eab5c1f1.jpg)